File: server/src/models/offsetData.js
const mongoose = require('mongoose');

const offsetDataSchema = new mongoose.Schema({
    Name: {
        type: String,    
    },
    Id: {
        type: String,
    },
    Brief: {
        type: String,
    },
    Min: {
        type: String,
    },
    Availability: {
        type: String,
    },
    Link: {
        type: String,
    },
    Image: {
        type: String,
    }
})
const OffsetData = mongoose.model('OffsetData', offsetDataSchema)

module.exports = OffsetData

File: client/carbonfit/lib/services/OffsetData.dart
import 'package:dio/dio.dart';
import 'package:fixyourprint/models/OffsetDataModel.dart';

class OffsetData {
  Dio dio = new Dio();
  var baseURL = "https://carbonfit-api.herokuapp.com/offset";
  List<OffsetDataModel> offsetDataList = [];

  Future<void> getData() async {
    var response = await dio.get('$baseURL');
    var data = response.data;

    if (response.statusCode == 200) {
      for (int i = 0; i < data.length; i++) {
        OffsetDataModel offsetDataModel = OffsetDataModel(
            Name: data[i]["Name"],
            Id: data[i]["Id"],
            Brief: data[i]["Brief"],
            Min: data[i]["Min"],
            Availability: data[i]["Availability"],
            Image: data[i]["Image"],
            Link: data[i]["Link"]);
        offsetDataList.add(offsetDataModel);
      }
    }
  }
}
